src/cpp/session/resources/grid/dtviewer.js

```diff
@@ -318,14 +318,20 @@
     }
 
     var escaped = escapeHtml(data);
-
+    
     // special additional rendering for cells which themselves contain data frames or lists:
     // these include an icon that can be clicked to view contents
     if (clazz === "dataCell" || clazz === "listCell") {
+      var truncated = escaped.endsWith("(...)");
+      if (clazz == "listCell" && truncated) {
+        escaped = escaped.replace(/[(][.][.][.][)]$/, "");
```

Comment:
This is fine but it took some puzzling for me to parse it out, perhaps `escaped.replace(/\(\.{3}\)$/, "");` might be clearer? (certainly debatable, I just more easily recognize the `\` as an escape symbol than `[]`)

Reply:
Yeah definitely, although I think I want to see if it's possible to not include "(...)" in the string to begin with, and mark it as truncated using some other mechanism. 

But otherwise, I agree that `\.{3}` is better.

Reply:
Yeah I was thinking that too, it seems kind of silly to add ` (...)` in the R code and then replace it in the JS code -- if you're going to replace it anyway, you might as well replace it with something that has a smaller possibility of user overlap (it doesn't seem that crazy that a user would have a value in a list column cell end with `(...)`)

Reply:
Using `[...]` and no longer styling it on the javascript side. 

I would prefer to add an attribute or something in `formatDataColumnList` but I haven't figured out how this would then be picked up on the other side.